src/platforms/common/enriching-events/scopes.mdx

```diff
@@ -93,7 +93,7 @@ You can also apply this configuration when unsetting a user at logout:
 To learn what useful information can be associated with scopes see
 [the context documentation](../context/).
 
-<PlatformSection notSupported={["android"]}>
+<PlatformSection>
 
 ## Local Scopes
 
```

Comment:
Well, we do have a bit of a discrepancy between the SDKs in this case:

[cocoa](https://github.com/getsentry/sentry-cocoa/blob/cd08c1734bd3891c89166bc9a54c1eb91177f6a0/Sources/Sentry/SentrySDK.m#L170) and [dart](https://github.com/getsentry/sentry-dart/blob/9157c9f3345817adef43420e12cf823c94da98ad/dart/lib/src/hub.dart#L84) don't capture exceptions that are thrown within the callback.

while [dotnet](https://github.com/getsentry/sentry-dotnet/blob/41acafb4b3bfced16b546dd7aa6900603b349ceb/src/Sentry/Internal/Hub.cs#L277) and [java](https://github.com/getsentry/sentry-java/blob/a88509c1b7b51bad58053b2457a79dd22fe29f70/sentry/src/main/java/io/sentry/Hub.java#L107) do because they invoke the callback within the try/catch of the respective `capture` methods

Reply:
I'll try to clarify

Reply:
I've created this https://github.com/getsentry/sentry-java/issues/2123 to keep track of it. It's not blocking this PR just notices this here and didn't wanna drop it.